docs/src/documentation/03-components/02-structure/tile/03-accessibility.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/tile/accessibility/
+---
+
+# Accessibility
+
+## Tile
+
+The Tile component has been designed with accessibility in mind.
+
+The component offers flexibility in terms of the HTML element used for the root node. By default, it will render a `div` element.
+
+### Expandable / Clickable
+
+The prop `expandable` makes the Tile clickable. In that case, assistive props like `ariaExpanded` are handled automatically.
+
+The component will render, by default, a `div` element with `role="button"` and `tabindex={0}`.
+
+This also happens when the `onClick` prop is defined.
+
+For that reason, one should refrain from using other interactive elements inside the Tile, like buttons or links, including on the `header` prop.
+
+### Href
+
+When the `href` prop is defined, the Tile will render an `a` element inside of the `div`.
+For the same reason as above, one should refrain from using other interactive elements inside the Tile.
+
+### HTML Title
+
+The `htmlTitle` prop can be used to add extra information to the Tile.
+
+It will be added as the `title` attribute to the root element (the `div` with `role="button"`) when the `expandable` is truthy or `onClick` is defined, or to the `a` element when the `href` prop is defined.
```

Comment:
> It will be added as the `title` attribute to the root element (the `div` with `role="button"`) when the `expandable` is truthy

Is this correct? When the `expandable` is truthy, we render `TileExpandable` which then passes `htmlTitle` to  `TileWrapper` but then inside the `TileWrapper`, `title` with `htmlTitle` is set only on the `a` element, not also on `WrapperComponent`. Or maybe looking at the wrong place 🙈

Reply:
That's true, I oversaw it being passed to the Wrapper inside the Expandable and not to the Slide, as it probably should 🤔 but yeah, I'd just rephrase it for now, and not change anything on the implementation